src/frontends/jax/src/op_table.cpp

```diff
@@ -94,6 +95,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops_jaxpr() {
             {"rsqrt", op::translate_rsqrt},
             {"reshape", op::translate_reshape},
             {"select_n", op::translate_select_n},
+            {"logistic", op::translate_logistic},
```

Comment:
please use `op::translate_1to1_match_1_input<...>` as for other activation operations. No need to have dedicate file with translator impl.

Reply:
sure, done